bundle/manifests/windows-instance-config-daemon_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -12,3 +12,4 @@ rules:
   - list
   - watch
   - get
+  - patch
```

Comment:
Do we need to remove this permission from WMCO?

Reply:
Nope, WMCO still patches the node with labels (worker label, BYOH label) and other annotations (public key hash, username anno)